Makefile

```diff
@@ -218,6 +218,15 @@ generate/kuma-cp/migrations:
 generate/gui: ## Generate gGOFLAGSo files with GUI static files to embed it into binary
 	GOFLAGS='${GOFLAGS}' go generate ./app/kuma-ui/pkg/resources/...
 
+generate/envoy-imports:
+	echo 'package envoy\n' > imports.go
+	echo '// Import all Envoy packages so protobuf are registered and are ready to used in functions such as MarshalAny.' >> imports.go
+	echo '// This file is autogenerated. run "make generate/envoy-imports" to regenerate it after go-control-plane upgrade' >> imports.go
+	echo 'import (' >> imports.go
+	go list github.com/envoyproxy/go-control-plane/... | grep "github.com/envoyproxy/go-control-plane/envoy/" | awk '{printf "\t_ \"%s\"\n", $$1}' >> imports.go
+	echo ')' >> imports.go
+	mv imports.go pkg/xds/envoy/imports.go
```

Comment:
Can you `>> pkg/xds/envoy/imports.go` and not have the mv. Or even better, have an env ENVOY_IMPORTS with the target name?

Reply:
for some reason I had problems with `>> full path`. Anyways, fixed now with env